content/shmem_my_pe.tex

```diff
@@ -30,7 +30,7 @@
 
 \apinotes{
     Each \ac{PE} has a unique number or identifier. As of \openshmem Specification
```

Comment:
Similarly, `\openshmem Specification 1.2` should be changed to `\openshmem[1.2]`. I think this is @jdinan's rule of "well, you touched it".

Reply:
@jdinan Should this change be fixed in #110 instead?

Reply:
Fixed in #110